mantidimaging/gui/windows/welcome_screen/view.py

```diff
@@ -91,3 +93,7 @@ def add_issues(self, issues_text: str) -> None:
                                         "  margin-top: 5px; "
                                         "}")
         self.issues_label.setText(issues_text)
+
```

Comment:
we can replace:
```python
def close_welcome_screen(self):
    self.parent().close()
```
with
```python
def close_welcome_screen(self):
    self.close()
    self.closed.emit()
```
so that we no longer call it's parent directly which I believe breaks encapsulation.